_drafts/circle-two-point-oh.md

```diff
@@ -0,0 +1,28 @@
+Notes from meeting in NYC:
+
+* We wanted to return early in the case of fast failure like rubocop and we
+  figured out how to hack it together in 1.0, but 2.0's workflows more closely
+  matched what we wanted to do.
+
+* start by describing the series of events.
+
+* do a gist of full config file, but also get the workflows part embedded.
```

Comment:
I will help find the example screenshots for both
1. Volt - to illustrate what we can achieve with 2.0 Workflow, like fan-in, fan-out, etc, and what we fail to achieve.
1. Lewitt-api - to illustrate the queueing problem, i.e. a simple sequential jobs could be interrupted and lead to significantly longer total time.